lib/react/jsx/babel_transformer.rb

```diff
@@ -4,17 +4,19 @@ module JSX
     class BabelTransformer
       DEPRECATED_OPTIONS = [:harmony, :strip_types, :asset_path]
       DEFAULT_TRANSFORM_OPTIONS = { blacklist: ['spec.functionName', 'validation.react', 'strict'] }
+
       def initialize(options)
+        return unless options.is_a?(Hash)
+
         if (options.keys & DEPRECATED_OPTIONS).any?
```

Comment:
:+1: from me to remove this. IIRC Babel::Transpilier raises an error on invalid options anyway, so this didn't actually gain us very much!